src/ui/server/controllers/insurance/business/company-details/index.test.ts

```diff
@@ -56,15 +60,59 @@ describe('controllers/insurance/business/companies-details', () => {
   });
 
   describe('get', () => {
-    it('should render the company-details template with correct variables', () => {
+    it('should render the company-details template with correct variables if application has populated exporterCompany data', () => {
```

Comment:
Can we wrap the conditional in a `describe`? Easier to read IMO e.g

```js
describe('When the application has populated exporterCompany data', () => {
  it('should....
});
```
Same below